bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -82,6 +94,10 @@
 		<label>Elapsed Time</label>
 		<description>Time elapsed in the program running on the appliance</description>
 		<category>Time</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
Ditto.

Reply:
Proposing `Calculation.Duration`

Reply:
Hmm, do we have something similar for other bindings? This is really just how long time has elapsed, so similar to elapsed time for a song being played on a media player.

Reply:
How about Progress

Reply:
How about Progress

Reply:
Status, Duration?

Reply:
See above